packages/vertica-nodejs/test/integration/client/connection-parameter-tests.js

```diff
@@ -3,13 +3,15 @@ const helper = require('../test-helper')
 const suite = new helper.Suite()
 const { Client } = helper.vertica
 
+/*
```

Comment:
Should leave a note here about why this is disabled

Reply:
Done